service/src/main/resources/HTMLTemplate.html

```diff
@@ -799,24 +803,28 @@ <h3 class="heading heading--two">Child {{ counter }}</h3>
             <dd class="data-list__definition">
               {% if child.value.livingSituation is not empty %}
                 <span class="u-block">{{child.value.livingSituation}}</span>
```

Comment:
Would you mind adding space between the braces.

Reply:
who called the code police 🙄

Reply:
;)